Feature/update admin lte

knoxfighter opened the pull request:
I've updated to admin-lte3 and tested the cards.

Know is the question, would you prefer bootstraps box, or the admin-lte-card? @mroote 
Both are used in the Mods-tab.
![grafik](https://user-images.githubusercontent.com/18680546/62248013-65e78600-b3e7-11e9-8ee2-c42756770b20.png)

mroote commented:
Nice! Lets go with the admin lte one but I really don't have a preference either way they both look fine.

knoxfighter commented:
AdminLTE3 is released now and i upgraded everything to work with adminLTE3 and bootstrap4.
The biggest changes are made to the categories (mostly used in the mods-window). Removed the old bootstrap4 boxes and added adminLTE3 cards instead.
Various Fixes for the new adminLTE Layouts.
Also updated all npm packages.

Additional fixed the up and down buttons in the Factorio Server Port Selection.

knoxfighter commented:
Since it was a few month, this PR got last updated, i decided to update the dependencies again. This time, the big thing was sweetalert2, with its new major version. All is done now and this PR can be merged after a review :smile:
